pkg/cap/cluster/container.go

```diff
@@ -0,0 +1,264 @@
+package cluster
+
+import (
+	"context"
+	"sync"
+
+	"capnproto.org/go/capnp/v3"
+	"capnproto.org/go/capnp/v3/server"
+	api "github.com/wetware/ww/internal/api/cluster"
+)
+
+var (
+	containerDefaultPolicy = server.Policy{
+		// HACK:  raise MaxConcurrentCalls to mitigate known deadlock condition.
+		//        https://github.com/capnproto/go-capnproto2/issues/189
+		MaxConcurrentCalls: 64,
+		AnswerQueueSize:    64,
+	}
+)
+
+type containerAnchor struct {
+	path   []string
+	client api.Container
+
+	release capnp.ReleaseFunc
+}
+
+func (ca containerAnchor) Name() string {
+	n := len(ca.Path()) - 1
+	if n >= 0 {
+		return ca.path[n]
+	}
+	return ""
+}
+
+func (ca containerAnchor) Path() []string {
+	return ca.path
+}
+
+func (ca containerAnchor) Ls(ctx context.Context) (AnchorIterator, error) {
+	fut, release := ca.client.Ls(ctx, func(a api.Anchor_ls_Params) error {
+		return nil
+	})
+	select {
+	case <-fut.Done():
+		results, err := fut.Struct()
+		if err != nil {
+			return nil, err
+		}
+		children, err := results.Children()
+		if err != nil {
+			return nil, err
+		} else {
+			return &containerAnchorIterator{path: ca.Path(), children: children, release: release}, err
+		}
+	case <-ctx.Done():
+		return nil, ctx.Err()
+	}
+}
+
+func (ca containerAnchor) Walk(ctx context.Context, path []string) (Anchor, error) {
+	if len(path) == 0 {
+		return ca, nil
+	}
+
+	fut, release := ca.client.Walk(ctx, func(a api.Anchor_walk_Params) error {
+		capPath, err := a.NewPath(int32(len(path)))
+		if err != nil {
+			return err
+		}
+		for i, e := range path {
+			if err := capPath.Set(i, e); err != nil {
+				return err
+			}
+		}
+		return nil
+	})
+	return containerAnchor{path: append(ca.Path(), path...), client: api.Container(fut.Anchor()), release: release}, nil
+}
+
+func (ca containerAnchor) Release(ctx context.Context) error {
+	if err := ca.client.Client.Resolve(ctx); err != nil {
+		return err
+	}
+	ca.client.Release()
+	return nil
+}
+
+func (ca containerAnchor) Set(ctx context.Context, data []byte) error {
+	c := api.Container{Client: ca.client.Client}
+	fut, release := c.Set(ctx, func(c api.Container_set_Params) error {
+		return c.SetData(data)
```

Comment:
Nit:  at first glance, this looks like you're calling `Container.SetData`.  Maybe rename variable to `ps`?  This would also be more consistent with the rest of the code-base.